_posts/loadbalancers/2024-03-08-create-a-ssl-loadbalancer.md

```diff
@@ -0,0 +1,361 @@
+connection to the webservers, the webservers itself need an ssl certificate as well. This is outside
+of the scope for this tutorial. 
+
+**Step 11**: Navigate to the `Health Monitor` tab by clicking on `Next`.  
+**Step 12**: Complete the following fields:  
+* **Name**: webserver-healthmonitor-http
+* **Type**: HTTP
+* **Max Retries Down**: 3
+* **Delay**: 5
+* **Max Retries**: 3
+* **Timeout**: 5
+* **HTTP Method**: GET
+* **Excepted Codes**: 200
+* **URL Path**: /
+* **Admin State Up**: Yes
+
+**Step 13**: Proceed to the `SSL Certificates` tab by clicking on `Next`.  
+**Step 14**: Add the appropriate certificate from the available certificates. It will be named
+`DomainName_complete_certificate`
+> Note: It is possible to add multiple certificates. The load balancer will use SNI to select the
+appropriate certificate
+
+
+**Step 15**: Initiate the creation of your load balancer by clicking on `Create Load Balancer`.  
+**Step 16**: Locate the load balancer you've just set up and click the small arrow beside it. From
+the dropdown menu, select `Associate Floating IP`.  
+**Step 17**: Select an available floating IP or choose the net-float pool, then confirm your choice
+by clicking on `Associate`.  
+
+**Step 18
+Finalize the deployment and start testing
+([Testing the loadbalancer](#testing-the-loadbalancer))
+
+## Creating the loadbalancer using the OpenStack CLI
+Now we can create the loadbalancer. We will create a loadbalancer with a listener, a pool and a
+healthmonitor. It is only possible to create the loadbalancer through 
+
+**Prerequisites**
+We should already have a bash variable set with the certificate location
+```bash
+octavia_certificate_url="https://keymanager.domain.tld:/v1/containers/uuid"
+```
+**Step 1**
+
+Gather the subnet uuid for the internal network:
+```bash
+openstack subnet list
+vip_subnet_uuid="uuid"
+```
+
+**Step 2**
+
+Create the loadbalancer
+```bash
+openstack loadbalancer create --name "webserver-loadbalancer" \
+--description "Loadbalancer for our webservers" --flavor Medium --vip-subnet-id "${vip_subnet_uuid}"
+```
+Make sure to save the returned `id` as variable, we need that later
+```bash
+lb_uuid=uuid
+```
+
+**Step 3**
+Create the listener
+```bash
+openstack loadbalancer listener create "${lb_uuid}" --name "webserver-listener-https" \
+--description "HTTPS Listener for our webservers" --protocol TERMINATED_HTTPS --protocol-port 443 \
+--default-tls-container-ref "${octavia_certificate_url}" 
+```
+
+Make sure to save the returned `id` as variable, we need that later
+```bash
+listener_uuid=uuid
+```
+
+**Step 4**
+
+Create the pool
+
+```bash
+openstack loadbalancer pool create --name "webserver-pool-http" \
+--description "HTTP Pool for our webservers" --protocol HTTP --lb-algorithm LEAST_CONNECTIONS \
+--listener "${listener_uuid}"
+```
+Make sure to save the returned `id` as variable, we need that later
+```bash
+pool_uuid="uuid"
+```
+
+
+**Step 5**
+
```

Comment:
Please remove the whitespace
![image](https://github.com/CloudTutorials/OpenStack-Docs/assets/31192098/eba406b0-9e16-46f5-8923-1e31e46bbd31)